packages/cdc-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,46 @@
+// Checkbox.tsx
+import { ChangeEvent, ReactNode, useState } from "react";
+import "./Checkbox.scss";
+
+type CheckboxProps = {
```

Comment:
Use `interface` until it needs to be `type`.

Reply:
Can you tell me more about this? I've been learning more TypeScript from Matt Pocock and I've started using types. Am I overlooking something?

Relevant References:
- [Article - Type vs Interface: Which Should You Use?](https://www.totaltypescript.com/type-vs-interface-which-should-you-use)
- [YouTube - TypeScript: Should you use Types or Interfaces?](https://www.youtube.com/watch?v=zM9UPcIyyhQ)
- [YouTube - Why use Type and not Interface in TypeScript](https://www.youtube.com/watch?v=Idf0zh9f3qQ)

Reply:
I agree too.